docs/kintoblock-builds.md

```diff
@@ -2,6 +2,47 @@
 title: Git Repository, Builds & CI
 ---
 
+## Branches & Tags
+
+KintoBlock is always in sync with your GitHub and Bitbucket repositories. If a user wants to add a branch to their repository, it can be directly added in KintoBlock. New branches can also be created in KintoBlock. At present, there is no option within KintoBlocks to modify the branches and tags.
```

Comment:
Currently there is no way to create a new branch within your kintoblock. Only the ability to find and link a new one within it from the repository. Seems a little confusing here.

Reply:
Yes, you are correct about current functionality - there is no option to create a branch via KintoHub. 

However, this is an upcoming feature which will be added to the product soon. I added the content and image based on this UI design: https://share.goabstract.com/408ee104-efdd-4e8a-bfd7-26fc10cfc680